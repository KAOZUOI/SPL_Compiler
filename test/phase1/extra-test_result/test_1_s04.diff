1,2c1
< Error type B at Line 1: Missing semicolon ';'
< Error type B at Line 4: Missing semicolon ';'
---
> Error type B at Line 3: Missing specifier
4c3
< Error type B at Line 11: Missing semicolon ';'
---
> Error type B at Line 9: Missing semicolon ';'
6,7c5,11
< Error type B at Line 29: Missing semicolon ';'
< Error type B at Line 37: Missing semicolon ';'
---
> Error type B at Line 17: Missing closing parenthesis ')'
> Error type B at Line 22: Missing closing parenthesis ')'
> Error type B at Line 23: Missing semicolon ';'
> Error type B at Line 31: Missing semicolon ';'
> Error type B at Line 32: Missing semicolon ';'
> Error type B at Line 42: Missing semicolon ';'
> Error type B at Line 46: Missing specifier
10c14
< Error type B at Line 59: Missing semicolon ';'
---
> Error type B at Line 60: Missing semicolon ';'
