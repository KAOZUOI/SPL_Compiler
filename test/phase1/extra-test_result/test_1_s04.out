Error type B at Line 1: Missing semicolon ';'
Error type B at Line 4: Missing semicolon ';'
Error type B at Line 5: Missing semicolon ';'
Error type B at Line 11: Missing semicolon ';'
Error type B at Line 12: Missing semicolon ';'
Error type B at Line 29: Missing semicolon ';'
Error type B at Line 37: Missing semicolon ';'
Error type B at Line 53: Missing semicolon ';'
Error type B at Line 55: Missing semicolon ';'
Error type B at Line 59: Missing semicolon ';'
